java/openmldb-taskmanager/src/test/java/com/_4paradigm/openmldb/taskmanager/TaskManagerClientTest.java

```diff
@@ -0,0 +1,22 @@
+package com._4paradigm.openmldb.taskmanager;
+
+import com._4paradigm.openmldb.taskmanager.client.TaskManagerClient;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+
+public abstract class TaskManagerClientTest {
+    @Ignore
+    @Test
+    public void TestManagerClient() throws Exception{
+        TaskManagerClient client = new TaskManagerClient("127.0.0.1:9995"); //要与taskmanager.properties里的配置一致
+//        client.stopJob(0);传入job id
+        client.printJobs();
+        client.showJobs();
+        client.importOnlineData("load data infile '/Users/4paradigm/IdeaProjects/OpenMLDB/java/openmldb-taskmanager/src/test/java/com/_4paradigm/openmldb/taskmanager/test.csv' " +
```

Comment:
Simplify the test and do not add local path in the code base.

Reply:
I get it